### PR TITLE
ID-1469 added new section for loaders

### PR DESCRIPTION
**Pull request by abodnar-svitla**
Added new section for loaders

### CODE REVIEW

**Review by tylerchilds — CHANGES_REQUESTED** [review]
When doing button small, the loader overlaps: 
screen shot 2018-01-25 at 2 25 00 pm

This should work for all button sizes. We should also utilize the existing `loader` class and move all of that into the styleguide, since this PR only solves the animated svg issue for buttons. For styling buttons, we should select like we do right now:

```
.loader.btn,
.loader[type=button],
.loader[type=submit],
button.loader{
```

**Comment by abodnar-svitla** [comment]
@tylerchilds, updated styles for loader

File: src/sass/components/_loader.scss [hunk 1]
```diff
@@ -5,60 +5,132 @@ Loader
...
+button.loader {
+  pointer-events: none;
+  position:relative;
+  &:after {
+    border: 2px solid transparent;
+    border-radius: 50%;
+    border-top: 2px solid white;
+    border-right: 2px solid white;
+
     position: absolute;
```
Comment (tylerchilds) [inline src/sass/components/_loader.scss]: let's remove 
```
position, top, bottom, right, margin
```

and make `margin-left: 10px`

File: src/sass/components/_loader.scss [hunk 2]
```diff
@@ -5,60 +5,132 @@ Loader
...
     top: 0px;
     bottom: 0px;
+    right: 10px;
+    margin: auto;
+
+    width: 10px;
+    height: 10px;
+    content: "";
+    display: inline-block;
+    animation: spin 1s linear infinite;
```
Comment (tylerchilds) [inline src/sass/components/_loader.scss]: Talked to Hunter, let's do this for easing function: `cubic-bezier(0.35, 0.55, 0.85, 0.95)`

File: src/sass/components/_loader.scss [hunk 3]
```diff
@@ -5,60 +5,132 @@ Loader
...
+    @include loader-small();
 
-/* Safari */
-@-webkit-keyframes spin {
-  0% { -webkit-transform: rotate(0deg); }
-  100% { -webkit-transform: rotate(360deg); }
+  }
+
+  &.btn--xsmall {
+    padding-right: 30px;
```
Comment (tylerchilds) [inline src/sass/components/_loader.scss]: with the above changes, we can remove these padding-right tweaks

**Review by tylerchilds — CHANGES_REQUESTED** [review]
A few minor things

**Comment by abodnar-svitla** [comment]
@tylerchilds, i fixed your comments, could you please check?

**Review by tylerchilds — CHANGES_REQUESTED** [review]
Final feedback: Just nudge the loaders with position: relative and top to make sure they're vertically centered with the text.

File: dist/scss/components/_loader.scss [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+.btn--small              - Small button with loader
+.btn--xsmall             - XSmall button with loader
+
+
+Styleguide 15.0
+
+*/
+
+@mixin loader-small {
+  padding-right: 30px;
```
Comment (tylerchilds) [inline dist/scss/components/_loader.scss]: we'll also want to remove this line

**Review by tylerchilds — APPROVED** [review]

**Pull request merged**